lib/jormungandr/exe/cardano-wallet-jormungandr.hs

```diff
@@ -273,7 +276,25 @@ genesisHashOption = optionT $ mempty
     <> help "Blake2b_256 hash of the genesis block, in base 16."
 
 -- | -- [ARGUMENTS...]
-extraArguments :: Parser [Text]
-extraArguments = some $ argument str $ mempty
+extraArguments :: Parser [String]
+extraArguments = many $ argument jmArg $ mempty
     <> metavar "[-- ARGUMENTS...]"
     <> help "Extra arguments to be passed to jormungandr."
+  where
+    jmArg = do
+        arg <- readerAsk
+        case validate arg of
+            Just err -> readerError err
+            Nothing -> pure arg
+    validate arg
+        | "--block-hash" `isPrefixOf` arg = Just $
```

Comment:
Shouldn't it be `--genesis-block-hash` ?

Reply:
Yes it should be, fixed.